From e66538f44c606d9f6c2ada9d78b310343e4386da Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Wed, 31 Aug 2016 15:13:28 +0500
Subject: [PATCH] json: correct layer lib paths

Rather than using a hardcoded ./ path for the libraries
we should use loose paths so the system could search
on its own when the library is required.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 layers/linux/VkLayer_core_validation.json      | 2 +-
 layers/linux/VkLayer_image.json                | 2 +-
 layers/linux/VkLayer_object_tracker.json       | 2 +-
 layers/linux/VkLayer_parameter_validation.json | 2 +-
 layers/linux/VkLayer_swapchain.json            | 2 +-
 layers/linux/VkLayer_threading.json            | 2 +-
 layers/linux/VkLayer_unique_objects.json       | 2 +-
 7 files changed, 7 insertions(+), 7 deletions(-)

Index: git/layers/linux/VkLayer_core_validation.json
===================================================================
--- git.orig/layers/linux/VkLayer_core_validation.json	2017-02-22 14:20:31.574212347 -0600
+++ git/layers/linux/VkLayer_core_validation.json	2017-02-22 14:29:15.000000000 -0600
@@ -3,7 +3,7 @@
     "layer" : {
         "name": "VK_LAYER_LUNARG_core_validation",
         "type": "GLOBAL",
-        "library_path": "./libVkLayer_core_validation.so",
+        "library_path": "libVkLayer_core_validation.so",
         "api_version": "1.0.39",
         "implementation_version": "1",
         "description": "LunarG Validation Layer",
Index: git/layers/linux/VkLayer_image.json
===================================================================
--- git.orig/layers/linux/VkLayer_image.json	2017-02-22 14:20:31.574212347 -0600
+++ git/layers/linux/VkLayer_image.json	2017-02-22 14:31:12.000000000 -0600
@@ -3,7 +3,7 @@
     "layer" : {
         "name": "VK_LAYER_LUNARG_image",
         "type": "GLOBAL",
-        "library_path": "./libVkLayer_image.so",
+        "library_path": "libVkLayer_image.so",
         "api_version": "1.0.39",
         "implementation_version": "1",
         "description": "LunarG Validation Layer",
Index: git/layers/linux/VkLayer_object_tracker.json
===================================================================
--- git.orig/layers/linux/VkLayer_object_tracker.json	2017-02-22 14:20:31.574212347 -0600
+++ git/layers/linux/VkLayer_object_tracker.json	2017-02-22 14:30:56.000000000 -0600
@@ -3,7 +3,7 @@
     "layer" : {
         "name": "VK_LAYER_LUNARG_object_tracker",
         "type": "GLOBAL",
-        "library_path": "./libVkLayer_object_tracker.so",
+        "library_path": "libVkLayer_object_tracker.so",
         "api_version": "1.0.39",
         "implementation_version": "1",
         "description": "LunarG Validation Layer",
Index: git/layers/linux/VkLayer_parameter_validation.json
===================================================================
--- git.orig/layers/linux/VkLayer_parameter_validation.json	2017-02-22 14:20:31.574212347 -0600
+++ git/layers/linux/VkLayer_parameter_validation.json	2017-02-22 14:30:44.000000000 -0600
@@ -3,7 +3,7 @@
     "layer" : {
         "name": "VK_LAYER_LUNARG_parameter_validation",
         "type": "GLOBAL",
-        "library_path": "./libVkLayer_parameter_validation.so",
+        "library_path": "libVkLayer_parameter_validation.so",
         "api_version": "1.0.39",
         "implementation_version": "1",
         "description": "LunarG Validation Layer",
Index: git/layers/linux/VkLayer_swapchain.json
===================================================================
--- git.orig/layers/linux/VkLayer_swapchain.json	2017-02-22 14:20:31.574212347 -0600
+++ git/layers/linux/VkLayer_swapchain.json	2017-02-22 14:30:33.000000000 -0600
@@ -3,7 +3,7 @@
     "layer" : {
         "name": "VK_LAYER_LUNARG_swapchain",
         "type": "GLOBAL",
-        "library_path": "./libVkLayer_swapchain.so",
+        "library_path": "libVkLayer_swapchain.so",
         "api_version": "1.0.39",
         "implementation_version": "1",
         "description": "LunarG Validation Layer",
Index: git/layers/linux/VkLayer_threading.json
===================================================================
--- git.orig/layers/linux/VkLayer_threading.json	2017-02-22 14:20:31.574212347 -0600
+++ git/layers/linux/VkLayer_threading.json	2017-02-22 14:30:24.000000000 -0600
@@ -3,7 +3,7 @@
     "layer" : {
         "name": "VK_LAYER_GOOGLE_threading",
         "type": "GLOBAL",
-        "library_path": "./libVkLayer_threading.so",
+        "library_path": "libVkLayer_threading.so",
         "api_version": "1.0.39",
         "implementation_version": "1",
         "description": "Google Validation Layer",
Index: git/layers/linux/VkLayer_unique_objects.json
===================================================================
--- git.orig/layers/linux/VkLayer_unique_objects.json	2017-02-22 14:20:31.574212347 -0600
+++ git/layers/linux/VkLayer_unique_objects.json	2017-02-22 14:30:14.000000000 -0600
@@ -3,7 +3,7 @@
     "layer" : {
         "name": "VK_LAYER_GOOGLE_unique_objects",
         "type": "GLOBAL",
-        "library_path": "./libVkLayer_unique_objects.so",
+        "library_path": "libVkLayer_unique_objects.so",
         "api_version": "1.0.39",
         "implementation_version": "1",
         "description": "Google Validation Layer"
